src/usr/share/gpgmailer/gpgkeyring.py

```diff
@@ -54,7 +56,11 @@ def __init__(self, gnupg_home):
 
             signed = self._is_key_signed(key)
 
+            for uid in key['uids']:
+                email_list.append(uid.split(" ").pop().strip(">|<"))
```

Comment:
Could you include an inline comment explaining what this line does. Please also include sample input in the inline comment.